### PR TITLE
kodiPackages.invidious: unstable-2022-11-28 -> 0.2.6

### PR DESCRIPTION
**Pull request by mguentner**
switch to maintained fork

https://github.com/petterreinholdtsen/kodi-invidious-plugin

which is also listed on kodi.tv:

https://kodi.tv/addons/nexus/plugin.video.invidious/

The original repository is now in archive mode:
https://github.com/TheAssassin/kodi-invidious-plugin/

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/kodi/addons/invidious/default.nix [hunk 1]
```diff
@@ -21,12 +18,16 @@ buildKodiAddon rec {
     inputstreamhelper
   ];
 
+  prePatch = ''
+    substituteInPlace Makefile --replace 'zip -r' '${zip}/bin/zip -r'
```
Comment (aanderse) [inline pkgs/applications/video/kodi/addons/invidious/default.nix:22]: i looked at the Makefile and it takes source to create a zip file... but we don't use the zip file

i don't think we need the Makefile in this case, do we?

**Review by aanderse — COMMENTED** [review]
> which is also listed on kodi.tv

I'm that case we should just use `fetchzip` like we do [here](https://github.com/NixOS/nixpkgs/blob/5550a85a087c04ddcace7f892b0bdc9d8bb080c8/pkgs/applications/video/kodi/addons/arrow/default.nix#L7), shouldn't we?

then we stay with the version the `kodi` promotes/provides

File: pkgs/applications/video/kodi/addons/invidious/default.nix (hunk 1 above)
Comment (mguentner) [reply pkgs/applications/video/kodi/addons/invidious/default.nix]: When `fetchzip` is used the `Makefile` is gone from `src` :)

**Comment by mguentner** [comment]
@aanderse Thanks for the review. PR is updated.

**Review by aanderse — CHANGES_REQUESTED** [review]
sorry i forgot to mention you should add `updateScript` to `passthru` so this will automatically be updated by our bot, like so:

```
passthru = {
  updateScript = addonUpdateScript {
    attrPath = "kodi.packages.arrow";
  };
};
```
aside from that all good to go

**Comment by mguentner** [comment]
@aanderse :heavy_check_mark:

**Review by aanderse — APPROVED** [review]
thanks!

**Pull request merged**